DEVELOPMENT.md

```diff
@@ -144,6 +144,24 @@ release: the actual image only gets pushed when there is a tag that
 corresponds to the version embedded in the source code. The
 Jenkinsfile ensures that.
 
+Release Checklist
```

Comment:
The title says Checklist but listed is more like "steps to execute", which for my eye is different thing. Checklist sounds more like things to check after something is already performed?

Reply:
It's normal for a "checklist" to also include things that need to be done. See for example https://wiki.mozilla.org/Releases/Checklist which contains tasks like "Email release-drivers" or "Push actual bits".

Reply:
ok, good as it is then.

Reply:
another thing about section header: should it be added in TOC?

Reply:
Yes, good catch. Fixed.

At some point we need to automate this. The kubernetes/enhancements repo has a script and check for it that we could copy.